thesis/inc/introduction/organization.tex

```diff
@@ -0,0 +1,9 @@
+\section{Organization}
+This document will be split into two main sections.
+The first section will discuss the NOX ECS product that was developed
+on a request by Suttung.
+The section will detail the product requirements, architecture,
+development issues, benchmarking, future work and a discussion on the longevity
+of some of the design choices taken in the development of the product.
+The second section will mainly cover the development process of the product.
+How did we plan our process, work methodology, tools used etc.
```

Comment:
We have a detailed description of all the parts in the first section, but just say etc... for the second one. Do we not yet know all the parts of the second section?